src/Frontend/src/components/configuration/PlatformLicense.vue

```diff
@@ -62,7 +62,7 @@ const loading = computed(() => {
                     </span>
                     <div class="license-expired-text" v-if="licenseStatus.isPlatformTrialExpired">Your license expired. To continue using the Particular Service Platform you'll need to extend your license.</div>
                     <div class="license-page-extend-trial" v-if="licenseStatus.isPlatformTrialExpiring && licenseStatus.isPlatformTrialExpired">
-                      <a class="btn btn-default btn-primary" href="https://particular.net/extend-your-trial?p=servicepulse" target="_blank">Extend your license&nbsp;&nbsp;<i class="fa fa-external-link"></i></a>
+                      <a class="btn btn-default btn-primary" :href="license.license_extension_url" target="_blank">Extend your license&nbsp;&nbsp;<i class="fa fa-external-link"></i></a>
```

Comment:
It may be the right time to create an ExternalLink component

Reply:
I'm missing a lot of context when in comes to Vue.js. Could you explain what would be the benefit of introducing an ExternalLink component?

Reply:
At the moment a dev needs to know all the styles, the external icon, the spaces between the icon and text ….
Those are all internal concerns. 
With something like
`<ExternalLink href=x />` they don’t need to know about those concerns. Also that makes it easier to update the external link look and feel in a single component. 
In webapps now a days with ui frameworks most html is abstracted away.

Reply:
Makes sense. I've captured that in an issue https://github.com/Particular/ServicePulse/issues/2117